.travis.yml

```diff
@@ -1,7 +1,37 @@
 language: node_js
 node_js:
-- '0.10'
+- '6'
+before_install:
+- wget https://s3-us-west-2.amazonaws.com/cdn.thegrid.io/caliper/libvips/libccv-0.1.1.tar.gz
```

Comment:
Nooo, what is this stuff? This is custom C++ libraries. Not cross-compiled for browser either. Not acceptable to include here.